packages/rn-tester/js/examples/TextInput/TextInputSharedExamples.js

```diff
@@ -152,13 +157,16 @@ class RewriteInvalidCharactersAndClearExample extends React.Component<
 
   constructor(props) {
     super(props);
-    this.state = {text: ''};
+    this.state = {text: '', focused: false};
   }
   render() {
     return (
       <View style={styles.rewriteContainer}>
         <TextInput
           testID="rewrite_clear_input"
+          accessible={true}
+          accessibilityState={{selected: this.state.focused}}
+          showSoftInputOnFocus={false}
```

Comment:
we may consider removing `showSoftInputOnFocus` before merging. Was added for demonstration purpose.

Reply:
I will move this example in AccessibilityExample.js